src/data/index.ts

```diff
@@ -14,6 +14,7 @@ import { getUnitAbilities, getUnitBoostCost } from "./units";
 import { getUpgradeAbilities } from "./upgrades";
 
 const ERROR_ON_FAILED_LOAD = false;
+export const ERROR_ON_MISSING_ABILITIES = false;
```

Comment:
We weren't erroring on failed load, but every card was null. This prevented me from "playing" heroes for my unit tests since their cards were nonexistent.

Reply:
I was initially like "this isn't ideal but I guess it's fine", but I think my stance is that I'm directly opposed to this for now. If you need to reference a card that doesn't have its effects loaded yet, I'd rather you just add your card to the appropriate load func (see Nautical Dog for an example)